### ISSUE TITLE
[Bug] Fine-tune job doesn't show actual default values for Batch size and Number of epochs

### ISSUE THREAD

**Issue by niksacdev** [microsoft-foundry/new-foundry-portal#48]
# Fine-tune job doesn't show actual default values for Batch size and Number of epochs

## Describe the Feedback
In the Fine-tune job configuration interface, when "Default" is selected for **Batch size** and **Number of epochs**, the UI does not display what the actual default values are. This creates confusion as users cannot see what parameters will be used for their fine-tuning job.

In contrast, the **Learning rate multiplier** field properly shows the actual value (e.g., "2") when configured, providing clear visibility into the parameter settings.

<img width="790" height="596" alt="Image" src="https://github.com/user-attachments/assets/82a6514e-b262-47f7-af1f-ad365adff1e1" />

**Observed Behavior:**
- Batch size: Shows "Default" (actual value unknown)
- Number of epochs: Shows "Default" (actual value unknown)
- Learning rate multiplier: Shows "2" (clear numeric value)
- Seed: Shows "974744287" (clear numeric value)

**Expected Behavior:**
All parameters should either:
1. Display the actual default numeric values instead of the word "Default", OR
2. Show the actual values in tooltips when hovering over "Default"

---

## Repro Steps
1. Go to AI Foundry workspace
2. Navigate to Fine-tuning section
3. Create or configure a fine-tune job
4. Observe the "Task parameters" section
5. Note that Batch size and Number of epochs show "Default" without revealing the actual values
6. Compare with Learning rate multiplier which shows the actual value "2"

## System Information
- **OS**: macOS
- **Browser**: [e.g., Chrome, Safari, Edge]
- **Version**: [Current browser version]

## Additional Context  
This issue impacts transparency and reproducibility of fine-tuning jobs. Users should be able to see all parameter values that will be used for training, especially when:
- Comparing different fine-tuning configurations
- Documenting training parameters for reproducibility
- Debugging fine-tuning performance issues
- Understanding why a particular fine-tuning job behaved a certain way

Suggested solutions:
1. Replace "Default" text with actual numeric values (e.g., "Default (32)" or just "32")
2. Add tooltips that display the actual default values when hovering
3. Include a documentation link explaining what the default values are and how they're determined
